honeycombio/honeycomb-network-agent
# maint: Remove unused kernel capabilities and update deployment examples

## Which problem is this PR solving?

Our system requirements were outdated in the README and deployments.

## Short description of the changes

- remove unused args from smoke test and quickstart deployments
- remove unused security capabilities (BPF and PERFMON) from deployments and README
- in smoke test deployment, wrap boolean environment variable for DEBUG in quotes so yaml treats it as a string
- remove references to ghcr secret used for downloading private package as it is no longer needed now that package is public

## How to verify that this has the expected result

The agent should continue to emit the same expected data as before, and these changes should have no impact on the agent's behavior.

- Following the quickstart continues to work, and the agent still captures the same data as it did before.
- For local development, update the smoke-tests/deployment to use the ghcr public image and run `make apply-agent` - it should work!
